src/CrossValidation.cpp

```diff
@@ -104,6 +105,46 @@ int CrossValidation::preIterationSetup(Scores& fullset, SanityCheck* pCheck,
     vector<Scores> myset(1, fullset);
     numPositive = pCheck->getInitDirection(myset, myset, pNorm, w_, testFdr_, initialSelectionFdr_);
   }
+
+  // Form cpos, cneg pairs per nested CV fold
+  cpCnTriple cpCnFold;
+  for (int set = 0; set < numFolds_; ++set) {
+    for (int nestedSet = 0; nestedSet < nestedXvalBins_; nestedSet++) {
+      if(!quickValidation_) {
```

Comment:
Can you fix the indentation for this block?

Reply:
Yeah, it should be fixed in the most recent commit.